.travis.yml

```diff
@@ -5,8 +5,16 @@ python:
   - 3.4
   - 3.5
   - 3.6
-  - pypy-5.6.0
-  - pypy3.3-5.5-alpha
+  - pypy
+  - pypy3
+matrix:
+    include:
+        - python: 3.7
+          dist: xenial
+        - python: 3.6
+          name: "Flake8"
+          install:  pip install -U flake8
+          script: flake8 --doctests src setup.py
```

Comment:
The rest of this file uses 2-space indents, can you please adjust to be consistent?

Reply:
Sure.